src/System.Text.Json/Directory.Build.props

```diff
@@ -6,5 +6,9 @@
     <StrongNameKeyId>Open</StrongNameKeyId>
     <IsNETCoreApp>true</IsNETCoreApp>
     <IsUAP>true</IsUAP>
+
+    <!-- Properties for source package -->
+    <_sourcePackageName>Microsoft.Bcl.Json.Sources</_sourcePackageName>
```

Comment:
I am not too familiar with the convention for item names within props/csproj files, but do we usually have `_camelCase` names or just `PascalCasing`?

Reply:
I generally depends on where are the properties used. We don't really have a convention, but we usually use _camelCase whenever we only use that property within one target or a single targets file. Think of it as the equivalent of a private member on a class kind of thing. Whereas we use PascalCasing for things that are used across multiple targets and multiple targets files, kind of like a public property.

Reply:
> but we usually use _camelCase whenever we only use that property within one target or a single targets file

Is this worth documenting this somewhere OR is this not really a rule that we follow strictly (nor do we want to)?